packages/application-generic/src/config/workers.ts

```diff
@@ -0,0 +1,59 @@
+enum WorkerEnum {
+  INBOUND_PARSE_MAIL = 'InboundParseMailWorker',
+  SUBSCRIBER_PROCESS = 'SubscriberProcessWorker',
+  STANDARD = 'StandardWorker',
+  WEB_SOCKET = 'WebSocketWorker',
+  WORKFLOW = 'WorkflowWorker',
+}
+
+const defaultConcurrency = process.env.WORKER_DEFAULT_CONCURRENCY
+  ? Number(process.env.WORKER_DEFAULT_CONCURRENCY)
+  : undefined;
+const defaultLockDuration = process.env.WORKER_DEFAULT_LOCK_DURATION
+  ? Number(process.env.WORKER_DEFAULT_LOCK_DURATION)
+  : undefined;
+
+interface IWorkerConfig {
+  concurrency: number;
+  lockDuration: number;
+}
+
+type WorkersConfig = Record<WorkerEnum, IWorkerConfig>;
+
+const workersConfig: WorkersConfig = {
+  [WorkerEnum.INBOUND_PARSE_MAIL]: {
+    concurrency: defaultConcurrency ?? 200,
+    lockDuration: defaultLockDuration ?? 900,
```

Comment:
```suggestion
    lockDuration: defaultLockDuration ?? 90000,
```

I think the old value is `90000` ?

Reply:
Good catch. I got confused with Active and Completed metrics workers.